build/ci/.azure-devops-project-template-tests.yml

```diff
@@ -0,0 +1,71 @@
+parameters:
+  vmImage: ''
+
+jobs:
+- job: Project Template Tests
+
+  pool:
+    vmImage: ${{ parameters.vmImage }}
+
+  dependsOn: VS_Latest
+
+  strategy:
```

Comment:
Doing a matrix for this is probably too heavy, as there are lots of nuget packages to restore. I'll probably be best to build each in a single job instance.